xonsh/completers/init.py

```diff
@@ -21,10 +20,10 @@ def default_completers():
     """Creates a copy of the default completers."""
     return collections.OrderedDict(
         [
-            ("python_mode", complete_python_mode),
             ("base", complete_base),
             ("completer", complete_completer),
             ("skip", complete_skipper),
+            ("end_proc_tokens", complete_end_proc_tokens),
```

Comment:
Should this be a separate completer? Or should it be part of the general completion framework?

Reply:
And does the completer ordering have any impact on how this executes?

Reply:
It needs to be a regular completer in the system to see the token.
Because `complete_base` will now work for stuff like `ls &&<TAB>`, we need to put it as the first completer, so I moved it now to be the first.